.github/workflows/container.yml

```diff
@@ -55,3 +56,19 @@ jobs:
             SHA=${{ github.sha }}
             URL=${{ github.server_url }}/${{ github.repository }}.git
             BRANCH=${{ github.ref_name }}
+
+  update-repo-description:
+    name: Update DockerHub Repository Description
+    runs-on: ubuntu-latest
+    if: github.ref == format('refs/heads/{0}', 'master')
```

Comment:
rel/stable may be the safest after us talking? This would limit the confusion people have on the :latest tagged container (which is the one they should be using)